networking_arista/ml2/arista_sec_gp.py

```diff
@@ -371,7 +371,8 @@ def _create_acl_rule(self, in_cmds, out_cmds, sgr, security_group_ips=None):
         Deals with multiple configurations - such as multiple switches
         """
         # Only deal with valid protocols - skip the rest
-        if not sgr or sgr['protocol'] not in SUPPORTED_SG_PROTOCOLS:
+        if not sgr or sgr['protocol'] not in SUPPORTED_SG_PROTOCOLS or (
+            sgr['ethertype'] not in SUPPORTED_SG_EHTERTYPES):
```

Comment:
We should also allow no ethertype specified. Ideally, in a way, that we can easily add IPv6.

Reply:
Thats true, the code atm only create ipv4 rules for both - ipv4 and ipv6 SGs

Reply:
Yes, but that check means, if `sgr['ethertype'] = None` then no security groups rules will be created, instead of only ipv4.

Reply:
Does this ever happen? but good objection, will default for ipv4 like it is done for None protocols

Reply:
Sure, just look at a default security group or a freshly created one.
It will have `protocol` set to `None` meaning any all protocols.